### PR TITLE
basic structure added

### PR DESCRIPTION
**Pull request by Sen-442b**
Button Components ✅
Alert Components ✅

Bugs: -
-dynamic text color feature class not working as intended
-inconsistency in color scheme

### CODE REVIEW

**Review by techieshubham30 — APPROVED** [review]

**Comment by techieshubham30** [comment]
Everything looks good.

**Review by msdeshmukh009 — APPROVED** [review]
Nice work!! This is looking nice and consistent. Those animations are also looking nice. Only one thing, in some places you are using direct colors instead of color variables. I hope you will work on that.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <!-- <link rel="stylesheet" href="./source/button.css" /> -->
```
Comment (mimansha-swarup) [inline index.html]: I will suggest to remove unnecessary comment

File: source/alert.css [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+    
+    font-size: 10px;
+
+    
+   
+}
+
+
+.alert-success{
+    background-color:#86efac;
```
Comment (mimansha-swarup) [inline source/alert.css]: use variable

File: source/alert.css [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+    animation-direction: alternate; */
+    
+
+
+}
+
+
+
+.alert:hover{
+    filter: brightness(1.2);
```
Comment (mimansha-swarup) [inline source/alert.css]: u have made a variable --hover-brightness 
why not use that

**Review by mimansha-swarup — COMMENTED** [review]
one more suggestion : you can make shadows of different intensity  and use that

**Comment by Sen-442b** [comment]
> Nice work!! This is looking nice and consistent. Those animations are also looking nice. Only one thing, in some places you are using direct colors instead of color variables. I hope you will work on that.

Thanks, and I'll work on that

File: source/alert.css (hunk 3 above)
Comment (Sen-442b) [reply source/alert.css]: > u have made a variable --hover-brightness why not use that

Thanks for pointing out Mimansa, I'll rectify it asap

**Comment by Sen-442b** [comment]
@mimansha-swarup Hey I made some updates, kindly check it once whenever you're free.

**Review by mimansha-swarup — APPROVED** [review]
looks good to me

**Pull request merged**